docs/docfx.json

```
@@ -50,7 +50,7 @@
     "overwrite": "_overwrites/**/**.md",
     "globalMetadata": {
       "_appTitle": "Discord.Net Documentation",
-      "_appFooter": "Discord.Net (c) 2015-2018 2.0.0-beta",
+      "_appFooter": "Discord.Net (c) 2015-2019 2.0.1",
```

Should we bump this to `2.0.2-dev`, since it was updated in db50bad?
```suggestion
      "_appFooter": "Discord.Net (c) 2015-2019 2.0.2-dev",
```